src/components/Header/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { HeaderPanel, SearchInput, ControlPanel, Toggle } from './styled';
+
+const Header = () => {
+  return (
+    <HeaderPanel data-testid="header">
+      <div className="ui icon button toggle-button">
+        <i className="bars icon"></i>
+      </div>
+      <SearchInput type="text" className="search-input" placeholder="Search..." />
+      <ControlPanel>
+        <Toggle className="ui toggle checkbox">
```

Comment:
Not a big fan of mixing styling alternatives. If you are using external classes, there's no point in Toggle being a styled component.